juju/juju
# Support for add-unit in new Kubernetes charms.

Add support for caas Application charms to have add-unit work.

## Checklist


## QA steps

Deploy Ben's snappass-test charm using he's guide.

juju add-model snappass

juju add-unit snappass-test 

\#  Verify that Kubernetes has scaled the deployment and that Juju status reports the new scale.

## Bug reference

https://bugs.launchpad.net/juju/+bug/1919421
